### DISCUSSION TITLE
能隐藏首页网盘列表或者添加首页访问密码吗

### DISCUSSION THREAD

**Discussion by wang-shi-jie** [AlistGo/alist#2163] [Q&A] [answered]
默认首页不用登录也能访问，在公司内部统一局域网内同事电脑能访问到私人的文件，请问如何通过验证后才能查看文件

**Comment by nini22P** [comment, marked as answer]
首先添加元信息，目录选根目录。
如果要隐藏就在隐藏那里填`.*`，把下面应用到子文件夹选项勾上。
如果要密码访问就在密码那里填密码，把下面应用到子文件夹选项勾上。

**Reply by starscen** [reply]
那么请教下，如果挂载的是某个网盘的根目录，但是想要隐藏掉这个网盘的某个文件夹，比如“我的相册”这个文件夹，该怎么写正则表达？

**Reply by nini22P** [reply]
> 那么请教下，如果挂载的是某个网盘的根目录，但是想要隐藏掉这个网盘的某个文件夹，比如“我的相册”这个文件夹，该怎么写正则表达？

直接填“我的相册”，想要更高级的匹配可以去学一下正则表达式

**Reply by starscen** [reply]
./115/我的相册
."115/我的相册"
./我的相册
/我的相册
我的相册


哪一种啊，实在不会

**Reply by starscen** [reply]
> > 那么请教下，如果挂载的是某个网盘的根目录，但是想要隐藏掉这个网盘的某个文件夹，比如“我的相册”这个文件夹，该怎么写正则表达？
> 
> 直接填“我的相册”，想要更高级的匹配可以去学一下正则表达式

挨个试了，是最后一个

**Comment by wang-shi-jie** [comment]
可以了，非常感谢。

**Comment by xsd116** [comment]
那如果那个文件夹是在我的相册下级呢     我的相册/下一个    这样吗？

**Reply by nini22P** [reply]
路径选”我的相册”，然后在隐藏一栏填你要隐藏的文件夹
